packages/oas-utils/src/helpers/omit-undefined-values.ts

```diff
@@ -0,0 +1,24 @@
+/**
+ * Removes undefined values from an object.
+ *
+ * Can be used as a transform function for any Zod schema.
+ */
+export const omitUndefinedValues = <T extends object>(data: T): T => {
+  // Handle arrays specially
+  if (Array.isArray(data)) {
+    return data.map((item) =>
```

Comment:
I thought isDefined might work here but we would mod it to allow null.

https://github.com/scalar/scalar/blob/main/packages/oas-utils/src/helpers/is-defined.ts

🤷🏾